coriolis/tests/api/middleware/test_auth.py

```diff
@@ -0,0 +1,286 @@
+# Copyright 2022 Cloudbase Solutions Srl
+# All Rights Reserved.
+
+from unittest import mock
+
+from oslo_middleware import request_id
+import webob
+
+from coriolis.api.middleware.auth import CoriolisKeystoneContext
```

Comment:
NIT: as per the agreed upon guidelines, we only import modules as a whole (and not module methods or classes). `Do not import objects, only modules (*)` (source: https://docs.openstack.org/hacking/latest/user/hacking.html#imports)
Therefore, please import `from coriolis.api.middleware import auth`, and change all `CoriolisKeystoneContext` occurances to `auth.CoriolisKeystoneContext`.